### PR TITLE
Move files around

### PR DESCRIPTION
**Pull request by SoyOscarRH**
Here is a proposal for modifying the structure of the webapp folder, here are some advantages:
* Here we use the advantage for index as a default file so we create files, so you as a developer can more easily see the work you are doing, as now each page have his own style and possible code without creating a mess
* The URLs are shorter, as we dont need .html anymore

### CODE REVIEW

File: fudge-ratio/src/main/webapp/about/index.html
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/index.html:5]: I THINK (OPINION), naming something ``x_style.css`` could be a little repetitive, as ``.css`` indicate that this is a style sheet, if I found ``x_code.css`` I will be really scared :v

File: fudge-ratio/src/main/webapp/calculator/index.html
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/calculator/index.html:49]: in general, we name things lowercase in a url, so is gmail.com not Gmail.com or facebook.com instead of FaceBook.com

File: fudge-ratio/src/main/webapp/calculator/index.html
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/calculator/index.html:51]: This is completely subjetive, naming is hard :v

File: fudge-ratio/src/main/webapp/index/index.html
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index/index.html:5]: I thinks is reflex the propose more

File: fudge-ratio/src/main/webapp/index/index.html
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index/index.html:26]: I think its a good practice to make all links to visit another pages absolute and delete the extension, giving you more flexibility

**Review by SoyOscarRH — COMMENTED** [review]
I made this PR basically using ``https://git-scm.com/docs/git-mv``, it is awesome :D 
Also, as I said at the beginning, this is a proposal, we could as a team choose to not merge this if someone feels an objection

**Comment by SoyOscarRH** [comment]
So clean now 
Screen Shot 2020-04-05 at 4 57 47 PM

File: fudge-ratio/src/main/webapp/about/index.html
Comment (yaaowusu00) [reply fudge-ratio/src/main/webapp/about/index.html:5]: I agree

**Review by yaaowusu00 — APPROVED** [review]
I was also thinking that we should organize the code more, so I really like what you did here! Thanks! I'm going to approve but if you want to wait to hear from Jerry and Giovanna before you merge thats fine

**Pull request merged**
